decidim/decidim
# Fix datetime fields on forms when the organization has a timezone specified

#### :tophat: What? Why?

Fix inconsistency when saving dates in the admin meeting component.

#### :pushpin: Related Issues

- Related to #12686
- Fixes #13576

#### Testing

1. Go to admin and change the organization timezone to something different than UTC.
2. Go to the configuration of a survey component and store a date with time for the "Answers accepted from/to" field.
3. Save and enter the form again
4. See the time is the same as the one you entered before.

### :camera: Screenshots

https://github.com/user-attachments/assets/684f35a4-51a6-45ee-95dd-0c842c9940da

:hearts: Thank you!
